### PR TITLE
parse SCTs in OCSPResponse

### PR DESCRIPTION
**Pull request by c0r0n3r**

### CODE REVIEW

File: src/cryptography/x509/extensions.py [hunk 1]
```diff
@@ -1483,6 +1480,16 @@ def __ne__(self, other):
         return not self == other
 
 
+@utils.register_interface(ExtensionType)
+class PrecertificateSignedCertificateTimestamps(SCTBase):
```
Comment (reaperhulk) [inline src/cryptography/x509/extensions.py]: I realize these are only being used to change the name of the class and the OID associated, but despite that I'm not wild about having a subclass. We have been very successful at not subclassing anything in this project. @alex do you have an opinion?

File: tests/x509/test_ocsp.py [hunk 2]
```diff
@@ -753,3 +753,26 @@ def test_invalid_serialize_encoding(self):
             resp.public_bytes("invalid")
         with pytest.raises(ValueError):
             resp.public_bytes(serialization.Encoding.PEM)
+
+    @pytest.mark.supported(
+        only_if=lambda backend: (
+            backend._lib.CRYPTOGRAPHY_OPENSSL_110F_OR_GREATER),
+        skip_message="Requires OpenSSL 1.1.0f+",
+    )
+    def test_single_extensions(self, backend):
```
Comment (reaperhulk) [inline tests/x509/test_ocsp.py]: Could you add one more test that asserts on the result of parsing this response when on OpenSSL < 1.1.0f?

File: src/cryptography/x509/extensions.py (hunk 1 above)
Comment (alex) [reply src/cryptography/x509/extensions.py]: I agree with you.

Comment (reaperhulk) [reply src/cryptography/x509/extensions.py]: @alex so what do we want to do here. Duplicate this class and its attendant tests?

Comment (alex) [reply src/cryptography/x509/extensions.py]: That's roughly what we do for all of the "this is an iterable" extensions, yes. If we want to think about reducing that duplication, there are things we could consider.

**Pull request closed without merging**